### PR TITLE
Pull in upstream updates from cisagov/skeleton-ansible-role

### LINKED ISSUE #2: The molecule tests for this role should actually test something
Right now there is just a dummy test.

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request pulls in upstream updates from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role).

## 💭 Motivation and Context ##

This Ansible role was quite out of date since it lacked a [lineage](https://github.com/cisagov/action-lineage) configuration.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] _All_ future TODOs are captured in issues, which are referenced in code comments.
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,16 @@ Here's how to use it in a playbook:
     - xfce
 ```
 
+## New Repositories from a Skeleton ##
+
+Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
+for step-by-step instructions on how to start a new repository from
+a skeleton. This will save you time and effort when configuring a
+new repository!
+
```
Comment (mcdonnnj) [inline README.md]: We're removing this on non-skeletons, right?

File: molecule/debian/molecule.yml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 ---
+# This molecule configuration file is suitable for testing Ansible
```
Comment (mcdonnnj) [inline molecule/debian/molecule.yml:2]: If this isn't going to symlink to `molecule-no-systemd.yml` or `molecule-with-systemd.yml`, does it makes sense to have those files symlinked here?

File: README.md (hunk 1 above)
Comment (dav3r) [inline README.md]: You missed removing this thang. ⚠️

File: molecule/redhat/molecule.yml [hunk 3]
```diff
@@ -1,4 +1,11 @@
 ---
+# This molecule configuration file is suitable for testing Ansible
```
Comment (mcdonnnj) [inline molecule/redhat/molecule.yml:2]: Same question about `molecule-no-systemd.yml` and `molecule-with-systemd.yml` symlinks here.

File: molecule/redhat/molecule.yml [hunk 4]
```diff
@@ -27,6 +32,35 @@ provisioner:
       # space and the slash.  This option converts those characters to
       # underscores when creating the group name.
       force_valid_group_names: always
+  inventory:
+    host_vars:
+      debian9:
```
Comment (mcdonnnj) [inline molecule/redhat/molecule.yml:37]: Since this appears to be RHEL/Fedora specific testing, should this remain?

**Review by mcdonnnj — APPROVED** [review]
Thanks for bringing another one into the light! 🔦 I had some questions about the molecule configuration in this repository below.

File: README.md (hunk 1 above)
Comment (dav3r) [reply README.md]: I see you took care of it in https://github.com/cisagov/ansible-role-xfce/pull/6/commits/66b35eefc5807fba626feda15325316613d00c38 👍

Comment (jsf9k) [reply README.md]: See commit 66b35ee.

File: molecule/debian/molecule.yml (hunk 2 above)
Comment (jsf9k) [reply molecule/debian/molecule.yml:2]: The point is to make the non-`default` scenarios look as much like the usual `default` scenario as possible.  That way it's easy to see where everything came from.  Also, if `molecule-no-systemd.yml` changes in the upstream skeleton then the changes are more likely to get noticed and moved into `debian/molecule.yml` this way.

File: molecule/redhat/molecule.yml (hunk 3 above)
Comment (jsf9k) [reply molecule/redhat/molecule.yml:2]: See [this comment](https://github.com/cisagov/ansible-role-xfce/pull/6#discussion_r540648406).

File: molecule/redhat/molecule.yml (hunk 4 above)
Comment (jsf9k) [reply molecule/redhat/molecule.yml:37]: Well it does no harm, and it reminds us that this file came from `default/molecule.yml`, so I left it in.

As with the [discussion here](https://github.com/cisagov/ansible-role-xfce/pull/6#discussion_r540648406), one could argue either way.  I think there is a slight benefit to leaving it in.

**Review by dav3r — APPROVED** [review]
Another one rides the (lineage) bus! 🚌

File: molecule/debian/molecule.yml (hunk 2 above)
Comment (mcdonnnj) [reply molecule/debian/molecule.yml:2]: That's reasonable. I just wanted to make sure we had intent behind leaving them. Thanks for the reply 👍

**Review by mcdonnnj — APPROVED** [review]
Thanks for getting this sorted and adding some testing along the way!

**Pull request merged**